src/Uno.UI.Runtime.Skia.Tizen/Tizen/System/Profile/TizenAnalyticsInfoExtension.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Tizen.System;
+using Windows.System.Profile;
+
+namespace Uno.UI.Runtime.Skia.Tizen.System.Profile
+{
+	internal class TizenAnalyticsInfoExtension : IAnalyticsInfoExtension
+	{
+		public TizenAnalyticsInfoExtension(object owner)
+		{
+		}
+
+		public UnoDeviceForm GetDeviceForm()
+		{
+			if (Information.TryGetValue<string>("http://tizen.org/feature/profile", out var profile))
+			{
+				if (profile != null)
+				{
+					profile = profile.ToUpperInvariant();
+					if (profile.StartsWith("M"))
```

Comment:
I'm curious, why check only the first letter?

Reply:
The profile check is based on Xamarin Essentials, but it seems the property does return the full name, so I have updated the logic accordingly 👍